test/test_maven_registry.d

```diff
@@ -0,0 +1,19 @@
+/+dub.sdl:
+dependency "vibe-d" version="~>0.8.3-alpha.1"
+versions "VibeNoSSL"
++/
+
+void main(string[] args)
+{
+	import std.conv, vibe.d;
+	immutable folder = readRequiredOption!string("folder", "Folder to service files from.");
+	immutable port = readRequiredOption!uint("port", "Port to use");
+	auto router = new URLRouter;
+	router.get("stop", (HTTPServerRequest req, HTTPServerResponse res){
+		res.writeVoidBody;
+		exitEventLoop();
+	});
+	router.get("*", folder.serveStaticFiles);
+	listenHTTP(text(":", port), router);
+	runApplication();
+}
```

Comment:
Might be worthwhile to look into whether the duplication from `test_registry.d` can be avoided.
Maybe a switch to make fallbacks optional?
Though OTOH it might be a good idea to check whether the retry logic actually works.

Reply:
You are right, I removed this file and use test_registry.d instead